tests/integration/targets/iosxr_acls/tests/cli/replaced.yaml

```diff
@@ -55,8 +61,7 @@
     - name: Assert that task was idempotent
       ansible.builtin.assert:
         that:
-          - result['changed'] == false
-          - result.commands|length == 0
+          - result.commands|length == 2
```

Comment:
how would a task be idempotent if the result.command length is not 0?
this is incorrect.

Reply:
apologies, i mistakenly considered it something else and thats why just overlooked at it, reverted!